tests/testthat/test-case.R

```diff
@@ -10,6 +10,25 @@ test_that("to_title creates one capital letter per word", {
 })
 
 test_that("to_sentence capitalizes just the first letter", {
-  x <- "This is a sentence."
   expect_identical(str_to_sentence("a Test"), "A test")
 })
+
+test_that("to_pascal converts to pascal case", {
+  expect_identical(str_to_pascal("This is a sentence."), "ThisIsASentence")
```

Comment:
I'd suggest testing cases the exercise each of the different regular expressions. I think that will be easier once you extract out a common "break into words" function.